tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import audio_utils
+from tensorflow_lite_support.python.task.audio.core.pybinds import audio_buffer
+
+
+class TensorAudio(object):
```

Comment:
@kinaryml TensorAudio should have two mode of internal storage.

* Mode 1: Numpy array. This is used for things like getting live data from sounddevice. I think you can reuse most of the logic [here](https://github.com/khanhlvg/tflite_raspberry_pi/blob/b1c1e3e38d80ebc8f59e8fc47d24493954c90161/audio_classification/audio_classifier.py#L71). In Numpy array storage mode, you'll need to implement a method to convert the internal `_buffer: ndarray` into an `AudioBuffer` that can be passed to the C++ AudioClassifier or AudioEmbedder.
* Mode 2: AudioBuffer instance. This is the mode where TensorAudio is created from a WAV file.

You can use a `is_from_file` flag similar to the TensorImage class to distinguish between the two modes.

Reply:
To implement the Numpy array storage mode, you can add the API that creates the sounddevice's InputStream like [this](https://github.com/khanhlvg/tflite_raspberry_pi/blob/b1c1e3e38d80ebc8f59e8fc47d24493954c90161/audio_classification/audio_classifier.py#L184) to the AudioClassifier.

Reply:
@khanhlvg How do I import the scipy dependency to try out Mode 1 in the bazel tests though?

Reply:
You can `pip install sounddevice` to have it installed in your local Python environment. `blaze run` will just use the locally installed sounddevice package and you don't need to add sounddevice into blaze dependencies.

It'll be similar to how you use on `numpy` in the project.

Reply:
@khanhlvg Thanks, got it!